frontend/src/components/ScheduledEventsRunHistoryTableCard/utils/index.ts

```diff
@@ -0,0 +1,26 @@
+import axios from 'axios';
+import * as localForage from 'localforage';
+import { ScheduledEventRunHistory } from '../../../types/scheduledEvents';
+import { api, localForageKeys } from '../../../utils';
+
+const BASEPATH = api.routes.FETCH_RUN_INSTANCES;
+
+export const fetchRunHistory = async (eventId: number): Promise<any> => {
```

Comment:
No unnecessary `any` ... avoid it like the plague and you'll master typescript types.